tests/src/tests.rs

```diff
@@ -40,24 +53,36 @@ const SVG_DIR: &str = "svg";
 const FONT_DIR: &str = "../assets/fonts";
 const ASSET_DIR: &str = "../assets";
 
+/// Arguments that modifies test behaviour
+///
+/// Specify them like this when developing:
+/// `cargo test --workspace --test tests -- --help`
 #[derive(Debug, Clone, Parser)]
 #[clap(name = "typst-test", author)]
 struct Args {
+    /// All the test that contains a filter string will be
+    /// run except if `--exact` is specified
     filter: Vec<String>,
-    /// runs only the specified subtest
+    /// runs only the specified test part
     #[arg(short, long)]
     #[arg(allow_hyphen_values = true)]
     subtest: Option<isize>,
+    /// runs only the test with the exact name specified in your command
+    /// ex: `cargo test --workspace --test tests compiler/bytes.typ -- --exact`
     #[arg(long)]
     exact: bool,
+    /// Update the reference images in `tests/ref`
     #[arg(long, default_value_t = env::var_os("UPDATE_EXPECT").is_some())]
     update: bool,
     #[arg(long)]
+    /// export the tests as pdf using Typst capabilities.
     pdf: bool,
     #[command(flatten)]
     print: PrintConfig,
     #[arg(long)]
-    nocapture: bool, // simply ignores the argument
+    nocapture: bool, // simply ignores the argument for backward compatibility
```

Comment:
This isn't for backwards compatibility, but rather because running `cargo test --workspace -- --nocapture` for the unit tests will otherwise fail the test runner. At least that's the idea, I'm not sure whether it would really still happen.